src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -1305,6 +1306,13 @@ namespace AirflowNetworkBalanceManager {
 					}
 				}
 			}
+			if ( MultizoneSurfaceData( i ).NodeNums( 2 ) == 0 && Surface( MultizoneSurfaceData( i ).SurfNum ).ExtBoundCond < 0 ) {
+				ShowSevereError( RoutineName + CurrentModuleObject + " = " + MultizoneSurfaceData( i ).SurfName );
+				ShowContinueError( "Outside boundary condition and object are " + cExtBoundCondition( Surface( MultizoneSurfaceData( i ).SurfNum ).ExtBoundCond ) + " and " + Surface( MultizoneSurfaceData( i ).SurfNum ).ExtBoundCondName + "." );
+				ShowContinueError( "The outside boundary condition must be either exposed to the outside or an adjacent zone." );
```

Comment:
@jasondegraw The change is OK with me. Is it possible to make it as:

ShowContinueError( "The outside boundary condition must be exposed to either the outside or an adjacent zone." );